gymnasium/envs/mujoco/mujoco_rendering.py

```diff
@@ -134,14 +139,22 @@ def close(self):
 class OffScreenViewer(BaseRender):
     """Offscreen rendering class with opengl context."""
 
-    def __init__(self, model: "mujoco.MjMujoco", data: "mujoco.MjData"):
-        width = model.vis.global_.offwidth
-        height = model.vis.global_.offheight
+    def __init__(
+        self,
+        model: "mujoco.MjMujoco",
+        data: "mujoco.MjData",
+        width: Optional[int] = None,
```

Comment:
Make `width` & `height` arguments non-optional, since they will be given my `mujocoEnv` class anyway

Reply:
this is misaligned with the init arguments of the `WindowViewer` and `MujocoRenderer`. perhaps this should be left as is?

Reply:
Sorry, I do not understand the issue, please elaborate

Reply:
in the current main branch, the `__init__` functions of the `WindowViewer` and `MujocoRenderer` classes also accepts `width` and `height` arguments. these arguments are optional and are `None` by default and are given sensible default values. I added `width: Optional[int] = None` and `height: Optional[int] = None` to be aligned with this way of thinking and use a sensible default that is the size of the offscreen framebuffer.

If we change to `width: int = 480` and `height: int = 480`, this will make `OffscreenViewer` the "odd man out" in this regard.

Reply:
Have you tested `width=None` it wouldn't work regardless cause `model.vis.global_.offwidth` wouldn't be set anyway

Reply:
The tests with `width=None` were removed in 4fdb196 but we can bring them back easily. Of course, this is your code so everything is your call. Should I bring back these tests or should I make the values non-optional?

Reply:
I see now what you mean. If we assume that everything goes through `MujocoEnv` then the width and height arguments will already be set.

I fixed this now in b5d8bc6. Note that this means that creating a `MujocoRenderer` object without `width` or `height` will cause an error. However, this will not happened with `MujocoEnv`.